utils/utils.go

```diff
@@ -126,7 +126,11 @@ func CustomizeRoute(route *routev1.Route, ba common.BaseComponent, key string, c
 				route.Spec.TLS.CACertificate = ""
 				route.Spec.TLS.Key = ""
 				route.Spec.TLS.DestinationCACertificate = ""
-				route.Spec.TLS.InsecureEdgeTerminationPolicy = ""
+				if *rt.GetInsecureEdgeTerminationPolicy() == routev1.InsecureEdgeTerminationPolicyRedirect {
```

Comment:
@arturdzm any particular reason why the InsecureEdgeTerminationPolicy was not being set for passthrough termination? It seems like Redirect and None are allowed values.

Reply:
@arturdzm please review the above question when you get a moment. Thanks.

Reply:
@yongja79 Need a nil check since InsecureEdgeTerminationPolicy could have not been specified in CR. Also, `Allow` is not a valid value for `passthrough`. The operator should throw a validation error (Reconcile should fail and output a relevant message).

Reply:
OpenShift UI does provide Redirect and None as options for passthrough

![passthrough](https://user-images.githubusercontent.com/18424008/146456309-efe56df9-9d17-45be-9be0-cc5a41c40bb9.png)